packages/webchat/src/main.tsx

```diff
@@ -416,40 +416,40 @@ export default inject(({ store }: { store: RootStore }) => ({
   displayWidgetView: store.view.displayWidgetView,
   setLoadingCompleted: store.view.setLoadingCompleted,
   sendFeedback: store.sendFeedback
-  // @ts-ignore
 }))(injectIntl(observer(Web)))
 
-type MainProps = { store: RootStore } & Pick<
-  StoreDef,
-  | 'config'
-  | 'initializeChat'
-  | 'botInfo'
-  | 'fetchBotInfo'
-  | 'sendMessage'
-  | 'setUserId'
-  | 'sendData'
-  | 'intl'
-  | 'isEmulator'
-  | 'updateTyping'
-  | 'setReference'
-  | 'updateBotUILanguage'
-  | 'hideChat'
-  | 'showChat'
-  | 'toggleBotInfo'
-  | 'widgetTransition'
-  | 'activeView'
-  | 'isFullscreen'
-  | 'unreadCount'
-  | 'hasUnreadMessages'
-  | 'showWidgetButton'
-  | 'addEventToConversation'
-  | 'clearMessages'
-  | 'updateConfig'
-  | 'mergeConfig'
-  | 'isWebchatReady'
-  | 'incrementUnread'
-  | 'displayWidgetView'
-  | 'resetUnread'
-  | 'setLoadingCompleted'
-  | 'dimensions'
->
+type MainProps = { store: RootStore } & WrappedComponentProps &
```

Comment:
Adding `& WrappedComponentProps` here allows removing the `@ts-ingore` at line 419